chevah_build

```diff
@@ -268,12 +314,12 @@ case $OS in
         export BUILD_LIBEDIT="no"
         ;;
     rhel5|sles11)
-        # Up-to-date cryptography not working on these...
+        # cryptography 1.3.4 is latest version with support for OpenSSL 0.9.8.
```

Comment:
I would also likes to see something like `PIP_LIBRARIES=OPENSSL_098_LIBRARIES`

Where OPENSSL_098_LIBRARIES is documented at the top, and consider whether we can use it on OSX.

Reply:
OK, no problem. Although this is the only place where these libs are used.